README.md

```diff
@@ -241,6 +241,37 @@ Linux or Mac using make
 These instructions are for building Simbody from source on either a Mac or on
 Ubuntu.
 
+#### Check the compiler version
+
+Simbody uses recent C++ features, that require a modern compiler.
+Before installing Simbody, check your compiler version with commands like that:
+
+- `g++ --version`
+- `clang --version`
```

Comment:
For consistency, could you make this either `g++`, `clang++`, or `gcc`, `clang`?